script/Registry.s.sol

```diff
@@ -39,6 +50,12 @@ uint256 constant AIRDROP_AMOUNT = 10_000e18; // 10k tokens
 uint256 constant MIN_DELAY = 3 days; // timelock for governor
 uint256 constant ORACLE_INTERVAL_PROD = 1 hours;
 
+// Vanity address params - use `cast create2` to find salt
+uint256 constant MAINNET_SALT_SYSTEMCOIN = 0;
+uint256 constant MAINNET_SALT_PROTOCOLTOKEN = 0;
+uint256 constant MAINNET_SALT_VAULT721 = 0;
+address constant MAINNET_CREATE2_FACTORY = address(0);
```

Comment:
Please create a new issue for yourself to research and identify / deploy the necessary create2 factory on Arbitrum mainnet. For next chapter

Reply:
I have a script ready to deploy one for mainnet.